apm-sniffer/config/agent.config

```diff
@@ -46,3 +46,12 @@ logging.file_name=${SW_LOGGING_FILE_NAME:skywalking-api.log}
 
 # Logging level
 logging.level=${SW_LOGGING_LEVEL:DEBUG}
+
+# Logging dir
+# logging.dir=${SW_LOGGING_DIR:""}
+
+# Logging max_file_size
+# logging.max_file_size=${SW_LOGGING_MAX_FILE_SIZE:300 * 1024 * 1024}
```

Comment:
Can we use an expression `300 * 1024 * 1024` here?

Reply:
I don't think so. Why open all this config? I think most users don't need to know this

Reply:
> I don't think so. Why open all this config? I think most users don't need to know this

Some users use a log collector and have their own acquisition rules. As a result, some customers need to change the location of the log output or other configurations